pontoon/base/signals.py

```diff
@@ -170,3 +170,29 @@ def assign_project_locale_group_permissions(sender, **kwargs):
 def create_user_profile(sender, instance, created, **kwargs):
     if created:
         UserProfile.objects.create(user=instance)
+
+
+@receiver(post_save, sender=Locale)
+def add_locale_to_system_projects(sender, instance, created, **kwargs):
```

Comment:
There's another `@receiver(post_save, sender=Locale)` above, I suggest we move this signal right after it.